sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -476,7 +476,7 @@ def __init__(self, label='ToList'):
     super(ToList, self).__init__(label)
 
   def expand(self, pcoll):
-    return pcoll | core.CombineGlobally(self.label, ToListCombineFn())
+    return pcoll | self.label >> core.CombineGlobally(ToListCombineFn())
```

Comment:
yeah I think we should remove label from __init__ of any derived class. I'll address that in another PR.